client/app/components/application.coffee

```diff
@@ -41,7 +42,14 @@ Application = React.createClass
 
     mixins: [
         TooltipRefesherMixin
-        StoreWatchMixin [SettingsStore, RefreshesStore, RouterStore, MessageStore, LayoutStore, AccountStore]
+        StoreWatchMixin [
+            SettingsStore
+            RefreshesStore
+            RouterStore
+            MessageStore
+            LayoutStore
+            AccountStore
+        ]
```

Comment:
`routerStore` contains all status of the application so that `MessageStore `, `LayoutStore `, or `AccountStore ` shouldn't be listened to. Can't you remove listener? If not why?